docs/make.jl

```diff
@@ -0,0 +1,32 @@
+using Documenter, StatsBase
+
+makedocs(
+    format = :html,
+    sitename = "StatsBase.jl",
+    # options
+    modules = [StatsBase],
+    pages = ["index.md",
+             "weights.md",
+             "means.md",
+             "scalarstats.md",
+             "robust.md",
+             "deviation.md",
+             "cov.md",
+             "counts.md",
+             "ranking.md",
+             "sampling.md",
+             "empirical.md",
+             "signalcorr.md",
+             "misc.md",
+             "statmodels.md"]   
+)
+
+deploydocs(
+    # options
+    repo = "github.com/JuliaStats/StatsBase.jl.git",
+    target = "build",
+    julia  = "nightly",
```

Comment:
Yeah, neither option is ideal. I've filed https://github.com/JuliaDocs/Documenter.jl/issues/526. In the meantime, I've changed the config file to use `0.6`, let's hope we don't forget to update it.